modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/BSTValidator.java

```diff
@@ -75,12 +76,11 @@ public Credential validate(Credential credential, RequestData requestData) throw
     public boolean authenticate(String binarySecurityToken) throws
```

Comment:
This method checks whether the BST resides in a cacheEntry, not in a user store as stated in the comment